apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -1,11 +1,49 @@
-/*global BridgeServiceMixin */
+/*global bridge,
+         BridgeServiceMixin,
+         BroadcastChannel,
+         Drafts,
+         streamClient,
+         Thread
+*/
 'use strict';
 
-if (!('BridgeServiceMixin' in self)) {
-  importScripts('/services/js/bridge_service_mixin.js');
-}
+/**
+ * Type description for the ConversationSummary object that is return from the
+ * Conversation service.
+ * @typedef {Object} ConversationSummary
+ * @property {number} id Unique identifier of the the conversation.
```

Comment:
Hmm, currently it should be number for both thread [1] and draft [2].

[1] http://mxr.mozilla.org/mozilla-central/source/dom/mobilemessage/interfaces/nsIDOMMozMobileMessageThread.idl#12
[2] https://github.com/mozilla-b2g/gaia/blob/ba33139c251108308cf882c152d9a7fff001bb83/apps/sms/services/js/drafts.js#L209